app.js

```diff
@@ -1,88 +1,863 @@
 // Task 1 - Notes for JS Evaluation Predictions
-
+/*
+ 4 + 4 / 2 ---> 6
+(2 + -7) * 3 ---> -15
+101 % 10 ---> 1
+12 - 4 % 3 ---> 11
+true && false ---> false
+true && !(false || false) ---> true
+!true && !(false || false) ---> false (surprised)
+'cat' + 'dog' ---> 'catdog'
+2 + 'pizza' ---> '2pizza
+2.5 * 'fish' ---> not a number
+5 >= 11 ---> false
+5 === 5.0 ---> true (surprised)
+7 !== 7.1 --->true
+5 + 5 > 8 --->true
+6 + 6 !== 12 --->false
+2 > 1 || false ---> true
+'true' === true ---> false
+10 % 2 === 0 ---> true
+21 % 2 === 0 ---> false 
+21 % 2 !== 0 ---> true 
+21 % 2 === 1 ---> true 
+12 % 3 === 0 ---> true
+9 % 3 === 0 ---> true
+14 % 3 === 0 ---> true
+'new york'[0] ---> 'n'
+'new york'[1] ---> 'e'
+'san francisco'[2 * 3] ---> 'a' but the second a (surprised)
+'engineering'[3].toUpperCase() ---> 'I"
+'engineering'.indexOf('G') ---> -1
+'engineering'.indexOf('neer') --->  4
+'engineering'.indexOf('r') > -1 ---> true
+'science'.indexOf('x') === -1 ---> true
+*/
 // List 3 expressions that surprised you or gave unexpected results, why?
+/*
+4 + 4 / 2 
+5 === 5.0
+'science'.indexOf('x') === -1
 
+*/
 // List 2 exprsssions you felt comfortable interpreting at a glance.
-
+/*
+'new york'[0] ---> 'n'
+6 + 6 !== 12 --->false
+*/
 // List 1 expression you are still 'shaky' on, why?
-
+//'science'.indexOf('x') === -1, how is it finding a position for x? but I think it's understands it as a negative regardless so it's more "falsy"
```

Comment:
Usually when you get back `-1` it means that what you are looking for does not exist!